src/test/resources/log4j.properties

```diff
@@ -8,6 +8,7 @@ log4j.appender.A1.layout=org.apache.log4j.PatternLayout
 log4j.appender.A1.layout.ConversionPattern=[%-5p] %d [%t] %x %c %M - %m%n
 
 log4j.logger.com.oltpbenchmark=INFO
+log4j.logger.com.oltpbenchmark.util.ThreadUtil=TRACE
```

Comment:
Do we want this to be part of the defaults? Not sure how verbose that would be.

Reply:
this is the test logging configuration.  ThreadUtil does not output much but what it does print is important to understanding its behavior.  TRACE on this file only adds a few lines to the output.  I'm happy to make a change here but don't see it as a big deal